### DISCUSSION TITLE
Project 3-Final project

### DISCUSSION THREAD

**Discussion by EleonoraMilea** [IBAHCM/RPiR#477] [Archive] [answered]
Hi @richardreeve and @soniamitchell, 
After creating the 4 specific functions and the general one in Project 3, I measured the indices of "tree.pop" expecting to see a lower diversity index as I increased the value of q (0,1,2, Inf). 

Overall, the value of D does show a decrease as q increases, however, my Simpson's diversity is smaller than the Berger-Parker index. 

Do you think that this is fine, or I am doing something wrong with one of the two functions? 

Thank you :)

**Comment by richardreeve** [comment, marked as answer]
@EleonoraMilea - I was about to respond saying everything sounded fine, but you've changed what you wrote here as I was typing! `D(2)` should be at least as big as `D(Inf)`. If this isn't true and you have any of these numbers less than 1, then it's probably because you're calculating the wrong thing - `1/D(2)`, for instance, which is Simpson's index, not `D(2)`.

**Reply by EleonoraMilea** [reply]
Hi @richardreeve,
thank you for your reply.

I noticed that the package vegan generates a higher Simpson's index compared to the one I am getting, and this value would also make more sense because it is larger than the Berger-Parker value. However, they use another formula (1-D), so it is not very useful as a reference for comparing my results. However, I asked other students and their results for the tree.pop data are the same, with a Simpson's index smaller than the Berger-Parker index. 

Thank you :)

**Reply by richardreeve** [reply]
Good - I'm glad that works for you.

Unfortunately there is a lot of confusion about diversity indices that use the word Simpson, which really doesn't help - various inverse/Simpson/index/diversity names exist - even when you use the same name it doesn't always mean exactly the same thing. The important thing is that in exercise 3, `D(2)` should be a number between 1 and the total number of species. The Simpson index/diversity should be a number between 0 and 1 (and is `1/D(2)`). Likewise `D(Inf)` is a number between 1 and the total number of species, and the Berger-Parker index should be a number between 0 and 1 (and is `1/D(Inf)`).

The end result of this is that "Simpson's index smaller than the Berger-Parker index", as you say, because `D(2) >= D(Inf)`.

**Reply by EleonoraMilea** [reply]
I can't believe it was that easy to understand, but good to know that my functions are working properly!

Thanks Richard :)
